### PR TITLE
Add rules to disable volume rewriting in RKE1

### PR DESCRIPTION
**Pull request by superseb**
@cmurphy Can you review this?

### CODE REVIEW

File: policy/centos7/rancher.te [hunk 1]
```diff
@@ -1,4 +1,4 @@
-policy_module(rancher, 1.0.0)
+policy_module(rancher, 1.0.1)
```
Comment (cmurphy) [inline policy/centos7/rancher.te:1]: I don't think this needs to be bumped, at least it never changes in rke2-selinux. @dweomer thoughts?

Comment (dweomer) [reply policy/centos7/rancher.te:1]: This version is disconnected from the artifacts that we release, which isn't great, but until we have a process to ensure such can be in sync we can avoid bumping it here (in-place upgrades seem to work just fine with same version).

**Review by dweomer — APPROVED** [review]
LGTM minus the module version bump

File: policy/centos7/rancher.te [hunk 2]
```diff
@@ -29,3 +29,24 @@ allow rke_logreader_t syslogd_var_run_t:dir read;
...
 allow rke_logreader_t var_log_t:file { getattr open read };
+
+########################
+# type rke_container_t #
+########################
+gen_require(`
+        type container_runtime_t, unconfined_service_t;
+        type container_log_t;
+        type syslogd_var_run_t;
+        type var_log_t;
```
Comment (cmurphy) [inline policy/centos7/rancher.te:40]: `syslogd_var_run_t` and `var_log_t` aren't used in the rules below so they shouldn't be needed

File: policy/centos7/rancher.te [hunk 3]
```diff
@@ -29,3 +29,24 @@ allow rke_logreader_t syslogd_var_run_t:dir read;
...
+        class dir { read search };
+        class file { open read };
+        class lnk_file { getattr read };
```
Comment (cmurphy) [inline policy/centos7/rancher.te:43]: Same with this, I don't think `lnk_file` is being used.

**Review by cmurphy — CHANGES_REQUESTED** [review]

File: policy/centos8/rancher.fc [hunk 4]
```diff
@@ -1 +1,2 @@
 /var/lib/rancher/rke(/.*)?                                              gen_context(system_u:object_r:container_var_lib_t,s0)
+/opt/rke(/.*)?                                                          gen_context(system_u:object_r:rke_opt_t,s0)
```
Comment (cmurphy) [inline policy/centos8/rancher.fc:2]: centos8 doesn't need `/etc/kubernetes(/.*)?`?

Comment (superseb) [reply policy/centos8/rancher.fc:2]: Newer `container-selinux` has `kubernetes_file_t`

**Review by cmurphy — APPROVED** [review]

**Comment by superseb** [comment]
@cmurphy @dweomer Can I merge & tag? `v0.2.testing.1`? If not, can you make a release?

**Comment by cmurphy** [comment]
New release https://github.com/rancher/rancher-selinux/releases/tag/v0.2-rc1.testing.1

**Pull request merged**